MSAL/test/app/ios/MSALTestAppAcquireTokenViewController.m

```diff
@@ -57,6 +57,8 @@
 #define TEST_EMBEDDED_WEBVIEW_MSAL 0
 #define TEST_EMBEDDED_WEBVIEW_CUSTOM 1
 
+static NSString *const kDeviceIdClaimsValue = @"{\"access_token\":{\"deviceid\":{\"essential\":true}}}";
```

Comment:
Just curious, in our code base, we have static variable names in the form of `s_***`, `k***` as well as underscore-concatenated capitals, do we have any preference or guidance? :)

Reply:
I don't :) But good callout to have an agreement.